### ISSUE TITLE
Capturing Packets Xterm window not appearing with Handshake Snooper

### ISSUE THREAD

**Issue by drjohannspector** [FluxionNetwork/fluxion#288]
----

#### Read the [wiki](https://github.com/FluxionNetwork/fluxion/wiki)?
yes

#### Do you want to request a *feature* or report a *bug*?
bug

#### What is the current behaviour?
not seeing the third xterm window appear. capturing data on channel X (airodump).
Shows up fine using other apps or previous versions of Fluxion

#### If the current behaviour is a bug, please provide the steps to reproduce it.
after scanning all channels, I select an AP. xterm window that displays attack/snoop every 30 seconds appears, along with the status of the attack itself. The xterm window for airodump does not appear. Although, everything seems to still work, I just can't see any MAC addresses, etc.

#### What do you think the expected behaviour should be?
after selecting handshake snooper and my attack, all 3 xterm windows should appear. 

#### Specs
[ BASH Info ]
GNU bash, version 4.4.12(1)-release (x86_64-pc-linux-gnu)
Copyright (C) 2016 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>

This is free software; you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
Path: /bin/bash

[ Interface (wlan0) Info ]
./diagnostics.sh: line 20: interface_physical: command not found
Device: Unknown
./diagnostics.sh: line 25: interface_driver: command not found
Driver: Unsupported
./diagnostics.sh: line 30: interface_chipset: command not found
Chipset: Unknown
Injection Test: Injection is working!

Ralink Technology, Corp. RT2870/RT3070

**Comment by MPX4132** [comment]
In order to properly terminate `airodump-ng`, which apparently requires receiving `SIGINT` (signal interrupt, aka, ctrl+c) to stop **gracefully**, fluxion needs to record the process identifier (pid) to be able to send it `SIGINT` when its capturing time is done.

The problem is that if we launch `airodump-ng` with `xterm` we can’t get `airodump-ng`‘s pid  because the last process to be launched was `xterm`.

What was happening before is that we were just killing `xterm`, but when it’s killled it sends `SIGTERM` (signal terminate) to whatever its running.

`airodump-ng` doesn’t like `SIGTERM` and when it is killed that way, it leaves the capture file corrupted sometimes, because it leave it complete.

TLDR:
I had to force `airodump-ng` to be executed in the background. It’s still there and running, but it’s running as a sub process of the `Handshake Snooper` arbiter daemon.

**Comment by MPX4132** [comment]
I’ll try to see if I can get it back to the foreground, on its own xterm somehow. I agree it helps with feedback.

**Comment by drjohannspector** [comment]
I feel naked without it

Appreciate anything you can do.

**Comment by MPX4132** [comment]
@drjohannspector It's back, check it out. It includes the bug fix too, I found a decently elegant solution.
